test/processor/Package.test.ts

```diff
@@ -627,5 +627,85 @@ describe('Package', () => {
       myPackage.optimize(processor);
       expect(profile.rules).toEqual([valueRule, extRule]);
     });
+    // suppressUrlAssignmentOnExtensions
+    it('should remove URL assignment rules on inline extensions on an extension', () => {
+      const extension = new ExportableExtension('MyExtension');
+      const containsRule = new ExportableContainsRule('extension');
+      containsRule.items.push({ name: 'foo' });
+      const assignmentRule = new ExportableFixedValueRule('extension[foo].url');
+      assignmentRule.fixedValue = 'foo';
+      extension.rules = [containsRule, assignmentRule];
+      const myPackage = new Package();
+      myPackage.add(extension);
+      myPackage.optimize(processor);
+      expect(extension.rules).toEqual([containsRule]);
+    });
+
+    it('should remove URL assignment rules on inline extensions on a profile', () => {
```

Comment:
While FSH/SUSHI don't expressly forbid inline extensions on profiles, I believe we've determined it really shouldn't be supported.

Reply:
While I agree that we have decided it is bad form, I'm not sure what I should change here. It is allowed in SUSHI without any error/warning. Do you think I should remove the processing of profiles altogether? Or just remove this test?

Reply:
How about this?  Go ahead and leave the logic/test -- but note in a comment that while SUSHI allows this, it is not actually valid FHIR and the IG Publisher will emit errors.  Does that work?